cellpack/tests/packing-configs/test_mesh_config.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "test_config",
+    "format": "simularium",
+    "inner_grid_method": "trimesh",
+    "live_packing": false,
+    "ordered_packing": false,
+    "out": "cellpack/tests/outputs/",
+    "overwrite_place_method": true,
+    "place_method": "spheresSST",
+    "save_analyze_result": true,
+    "load_from_grid_file": false,
+    "show_progress_bar": true,
+    "show_grid_plot": true,
+    "spacing": null,
+    "parallel": false,
+    "use_periodicity": false,
+    "show_sphere_trees": true
+}
```

Comment:
how is this different than the other test configs we have?

Reply:
This config re-creates the grid by default, and also shows the grid information in the simularium file. This is useful for debugging recipes involving meshes.